data_steward/cleaners/clean_cdr_engine.py

```diff
@@ -42,11 +51,28 @@ def clean_dataset(project=None, dataset=None, statements=None):
     failures = 0
     successes = 0
     for statement in statements:
-        rule_query = statement.format(project=project, dataset=dataset)
+        query = statement.get(cdr_consts.QUERY, '')
+        rule_query = query.format(project=project, dataset=dataset)
+
+        legacy_sql = statement.get(cdr_consts.LEGACY_SQL, False)
+        destination_table = statement.get(cdr_consts.DESTINATION_TABLE, None)
+        retry = statement.get(cdr_consts.RETRY_COUNT, bq_consts.BQ_DEFAULT_RETRY_COUNT)
+        disposition = statement.get(cdr_consts.DISPOSITION, 'WRITE_EMPTY')
```

Comment:
I've created a patch from your branch of my suggested changes.  Let me know if you want to look at it so you can apply it yourself to see how the code looks.  I'll discuss my suggestions here.  

I've noticed we use the write disposition strings frequently and now we're doing it in multiple places.  I think we should add WRITE_EMPTY, WRITE_TRUNCATE, and WRITE_APPEND to our constants/bq_utils.py module.  You've used the lowercase variants for the spellings in both query files.  BigQuery defines these as upper cased constants.  BQ probably won't recognize lower cased variants.  By adding these definitions to the constants file, we can import them and use them instead of writing each one and potentially introducing an error through misspellings.  

The patch addresses the write dispositions in bq_utils.py, constants/bq_utils.py, cleaners/id_deduplicate.py, and cleaners/null_invalid_foreign_keys.py

Reply:
can you push the changes to this branch

Reply:
I can.  But I wasn't sure if you would want me to push the changes or let you review my suggestions first.  I will push them if you want.

Reply:
I pushed my suggestions to the branch.  Look them over and let me know if you've got questions about why a change was suggested.